### ISSUE TITLE
web query times out for some spaxel queries

### ISSUE THREAD

**Issue by havok2063** [sdss/marvin#567]
**Describe the bug**
During the crash day, some attempts at doing a web spaxel query timed out.  Returns a "504 Gateway Time-out" error.  Could be a legit error with the spaxelprop datamodel or could be a long-running query that simply times out.  

**To Reproduce**
Try queries `spaxelprop.emline_gflux_ha_6564 > -1` or `spaxelprop.nii_to_ha > 1 AND spaxelprop.emline_gvel_ha_6564 > 150`

**Expected behaviour**
Either a successful query or more informative error message.  Needs testing in web and tools.

**Comment by havok2063** [comment]
Closing this.  May no longer be relevant. Can open a new ticket if needed.

**Comment by heretzp** [comment]
Hello, I am experiencing a 504 error on my query. I am a new marvin user, so my query may just be too large. Any help would be much appreciated, thank you!
![image](https://github.com/user-attachments/assets/871f8e00-dcba-474a-a6af-a232e05f5e2f)

**Comment by heretzp** [comment]
An update about the behavior: it seems that return_params is causing my 504 issue. I simplified my filter to just include nsa.z<0.1. I was able to run and print the query results without additional parameters (ie, did not include a return_params statement). 
I then added one additional parameter to return_params (not from the spaxelprop datamodel as you suggest above), nsa.extinction in this case, and am still met with the 504 timeout:
![image](https://github.com/user-attachments/assets/b92446ae-2d67-4024-8845-22ada7f3d28e)
